packages/sanity/src/core/studio/components/navbar/search/contexts/search/SearchProvider.tsx

```diff
@@ -24,15 +25,29 @@ import {initialSearchState, searchReducer} from './reducer'
 interface SearchProviderProps {
   children?: ReactNode
   fullscreen?: boolean
+  /**
+   * if provided, then it means that the search is being done from within a specific perspective
+   */
+  perspective?: string
+  /**
+   * if provided, then ids should be checked against this list
+   */
+  listOfDocumentIds?: string[]
```

Comment:
What does "checked against this list" mean? Also could it be named `documentIds`? (List/array is clearly communicated via the type already)

Reply:
Yep, good point! 
I'll rename!